frontend/src/App.tsx

```diff
@@ -1,10 +1,10 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react";
```

Comment:
I don't like having to use semicolons. Change Prettier and/or eslint so we don't need semicolons.